flake.nix

```diff
@@ -485,6 +483,55 @@
               ```
             '';
           };
+
+          frontend-static = mkFrontendStatic {
+            subsquidEndpoint = "http://localhost:4350/graphql";
+            picassoEndpoint = "ws://localhost:9988";
+            kusamaEndpoint = "ws://localhost:9944";
```

Comment:
it is rococo endpoint

Reply:
It can be kusama or even polkadot, must probably be renamed relaychain but this how FE named it